### PR TITLE
Adding CI for oVirt/csi-driver-operator

### PR DESCRIPTION
**Pull request by rgolangh**

### CODE REVIEW

**Comment by rgolangh** [comment]
@sandrobonazzola PTAL

File: ci-operator/config/oVirt/csi-driver-operator/OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- bennyz
+- rgolangh
```
Comment (sandrobonazzola) [inline ci-operator/config/oVirt/csi-driver-operator/OWNERS:6]: this is CI, I would also add Gal Zaidman to reviewers / approvers

File: ci-operator/jobs/oVirt/csi-driver-operator/OWNERS [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- bennyz
+- rgolangh
```
Comment (sandrobonazzola) [inline ci-operator/jobs/oVirt/csi-driver-operator/OWNERS:6]: same here, I would add Gal Zaidman.

**Comment by rgolangh** [comment]
done

File: ci-operator/config/oVirt/csi-driver-operator/OWNERS (hunk 1 above)
Comment (rgolangh) [reply ci-operator/config/oVirt/csi-driver-operator/OWNERS:6]: done

**Comment by rgolangh** [comment]
> I think you will also want the approve plugins, which needs to be enabled in
> `core-services/prow/02_config/_plugins.yaml` for this repo

@alvaroaleman I think the approve plugin is set for the whole oVirt org. I did add few more plugins. (copy pasted from oVirt/csi-driver) section.

**Pull request merged**